.github/release_template.md

```diff
@@ -0,0 +1,46 @@
+<!--
+This is the template to release a new version on the App Store
+-->
+
+Release a new version
+## TASKS:
+
+Git & Code
+- [ ] [GIT] Create branch release/[major].[minor].[patch] (freeze the code)
+- [ ] [DEV] Update version number [major].[minor].[patch]
+- [ ] [DIS] Update changelogs:
+     - [ ] [DEV] Update changelog https://github.com/owncloud/ios-app/blob/master/CHANGELOG.md
+     - [ ] [GIT] Add the new release on GitHub https://github.com/owncloud/ios-app/releases
```

Comment:
Probably makes sense to only add the release after QA and docs have been updated as well, so it represents the final release. So, maybe move this to the end of this section.